src/Components/Activities/Detail/Detail.js

```diff
@@ -22,13 +22,10 @@ const Detail = () => {
 
   useEffect(() => {
     getActivity(id)
-      .then((response) => {
-          setActivity(response.data.data);
+      .then((gettedActivity) => {
```

Comment:
it should be something like activityData or activityResponse because gettedActivity is strange

Reply:
Change applied